integration/environment/environment.go

```diff
@@ -52,7 +55,7 @@ func NewEnvironment() *Environment {
 			Fs:         afero.NewOsFs(),
 		},
 		Machine: Machine{
-			pollTimeout:  30 * time.Second,
+			pollTimeout:  60 * time.Second,
```

Comment:
Is it necessary to increase the timeout?

Reply:
nope, forgot to switch it back 👍